packages/terminal/src/browser/base/terminal-widget.ts

```diff
@@ -106,4 +106,9 @@ export interface TerminalWidgetOptions {
      * Terminal attributes. Can be useful to apply some implementation specific information.
      */
     readonly attributes?: { [key: string]: string | null };
+
+    /**
+     * Initial message to be displayed while terminal is not started.
+     */
+    readonly loadingMessage?: string;
```

Comment:
We should test reload page reload. What would you expect should happen? We try to preserve terminals. I assume terminal should appear immediately with loading message till connection to the task process is not recovered. We need ways to test it with the vanilla Theia.